MobilliumQRCodeReader.podspec

```diff
@@ -8,35 +8,20 @@
 
 Pod::Spec.new do |s|
   s.name             = 'MobilliumQRCodeReader'
-  s.version          = '0.1.0'
+  s.version          = '1.0.0'
   s.summary          = 'A short description of MobilliumQRCodeReader.'
 
-# This description is used to generate tags and improve search results.
-#   * Think: What does it do? Why did you write it? What is the focus?
-#   * Try to keep it short, snappy and to the point.
-#   * Write the description between the DESC delimiters below.
-#   * Finally, don't worry about the indent, CocoaPods strips it!
-
   s.description      = <<-DESC
-TODO: Add long description of the pod here.
+  MobilliumQRCodeReader is a customisable qr code reader also you can read qr code from gallery image.
                        DESC
 
   s.homepage         = 'https://github.com/mrtcelebi/MobilliumQRCodeReader'
-  # s.screenshots     = 'www.example.com/screenshots_1', 'www.example.com/screenshots_2'
   s.license          = { :type => 'MIT', :file => 'LICENSE' }
   s.author           = { 'mrtcelebi' => 'mrttcelebi@gmail.com' }
   s.source           = { :git => 'https://github.com/mrtcelebi/MobilliumQRCodeReader.git', :tag => s.version.to_s }
```

Comment:
sources must be 'https://github.com/mobillium/MobilliumQRCodeReader.git'

Reply:
fixed.